docs/user/grading.rst

```diff
@@ -34,3 +46,67 @@ This means the students can track their relative performance during the semester
     .. figure:: grading/course_statistics_attainable.png
        :alt: Bonus Grade in Course Statistics Page
        :align: center
+
+Bonus
+-----
+Instructors can create bonus configurations for exams with ``Grade`` type grading keys by clicking on |bonus| button in the exam detail page.
+A bonus configuration maps the grade received from a bonus source, which can be a course or another exam, as an improvement to the final exam grade.
+In order to configure a bonus, an instructor needs to choose appropriate values for the fields below:
+
+1. **Bonus strategy** defines how the defines the grade obtained from the bonus source will affect the final exam. Artemis currently supports 2 strategies:
+
+  - *Grades*: First, calculates the target exam grade. Then, applies the bonus to that.
+  - *Points*: First, applies the bonus to the student's points. Then, calculates the final grade by matching the resulting points to the target exam's grading key.
+
+2. **Discreteness** (Only applicable if *grades* bonus strategy is selected) specifies how to combine the bonus grade with the exam grade. Currently only the first discreteness option is implemented.
+
+  - *Continuous*: Applies bonus arithmetically to the student's grade. Final grade can be any numeric value between the best and the worst grade step values (e.g. from 1.3 to 1.2).
+  - *Discrete*: (Not available yet) Bumps the student's grade to a better grade step. Final grade will be one of the grade steps in the target exam (e.g. from 1.3 to 1.0 or from B to A).
+
+3. **Calculation** defines the sign of the operation to indicate if it is an addition or subtraction.
+
+  - *−* (Default option for *grades*): Subtracts bonus from target exam's grades/points. Prefer this when lower means better.
+  - *+* (Default option for *points*): Adds bonus to target exam's grades/points. Prefer this when higher means better.
+
+4. **Bonus source** is the course or exam whose grade . The dropdown lists courses and exams with ``Bonus`` type grading keys if the current user is an instructor of it.
```

Comment:
There is a typo.

Reply:
Thanks, I fixed it in https://github.com/ls1intum/Artemis/pull/6087/commits/f22b3482298322889889b9ae3493ac8f7ed6fc83